### ISSUE TITLE
Expected title in test case is wrong

### ISSUE THREAD

**Issue by fintara** [mozilla/readability#452]
Hello,
In test case "engadget", expected title is "A console that keeps up with gaming PCs", but the proper one should be "Xbox One X review: A console that keeps up with gaming PCs".

[https://github.com/mozilla/readability/blob/master/Readability.js#L386](Here) it checks whether the title can be found in any of h1 or h2 tags. It can be found, but in the source there is an extra space at the end, hence `match` is false.

So maybe `heading.textContent` should be trimmed when comparing with `curTitle` (or `getInnerText` should be used).

**Comment by gijsk** [comment]
Yeah, we should call `.trim()` on both strings before comparing them, I expect.
